build/assembly/jboss-as7/dist.xml

```diff
@@ -5,6 +5,7 @@
   
   <formats>
     <format>zip</format>
+    <format>dir</format>
```

Comment:
Leaves Teiid jboss dist unpacked in target folder for use in the server assembly that runs later.